rspack.config.ts

```diff
@@ -4,21 +4,13 @@
 import fs from 'fs';
 import path from 'path';
 
-import {WebpackReactSourcemapsPlugin} from '@acemarke/react-prod-sourcemaps';
+import rspack from '@rspack/core';
 import CompressionPlugin from 'compression-webpack-plugin';
-import CopyPlugin from 'copy-webpack-plugin';
-import CssMinimizerPlugin from 'css-minimizer-webpack-plugin';
-import ForkTsCheckerWebpackPlugin from 'fork-ts-checker-webpack-plugin';
-import MiniCssExtractPlugin from 'mini-css-extract-plugin';
+import ReactRefreshRspackPlugin from '@rspack/plugin-react-refresh';
 import webpack from 'webpack';
```

Comment:
Can drop the webpack import?

Reply:
There are webpack plugins that are not available on rspack that we rely on.